### PR TITLE
adds /controllers list to management and client API

### PR DESCRIPTION
**Pull request by andrewpmartinez**
Most of this PR is generated. I added comments on the two main files that were changed.

### CODE REVIEW

File: management.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ info:
   license:
```
Comment (andrewpmartinez) [inline management.yml:15]: Focus on this file.

File: client.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ info:
   license:
```
Comment (andrewpmartinez) [inline client.yml:15]: Focus on this file.

File: management.yml [hunk 3]
```diff
@@ -216,6 +216,8 @@ paths:
     get:
       security:
       - ztSession: []
+      - oauth2:
+        - openid
```
Comment (andrewpmartinez) [inline management.yml:220]: The OIDC was missing on these endpoints, causing OIDC bearer tokens to not work for auth.

File: client.yml [hunk 4]
```diff
@@ -280,6 +280,98 @@ paths:
...
+  /controllers:
+    get:
+      security:
+      - ztSession: []
+      - oauth2:
+        - openid
+      description: Retrieves a list of controllers
+      tags:
+      - Controllers
+      summary: List CAs
```
Comment (dovholuknf) [inline client.yml:292]: List CAs ?

File: client.yml [hunk 5]
```diff
@@ -280,6 +280,98 @@ paths:
...
+                    property: fooField3
+                  field: (root)
+                  message: '(root): fooField3 is required'
+                  type: required
+                  value:
+                    fooField: abc
+                    fooField2: def
+                causeMessage: schema validation failed
+                code: COULD_NOT_VALIDATE
+                message: The supplied request contains an invalid document
```
Comment (dovholuknf) [inline client.yml:333]: wording sounds off in some ways. "The supplied document is invalid" or "contains an invalid field"? or meh...

File: client.yml [hunk 6]
```diff
@@ -280,6 +280,98 @@ paths:
...
+                requestId: ac6766d6-3a09-44b3-8d8a-1b541d97fdd9
+              meta:
+                apiEnrollmentVersion: 0.0.1
+                apiVersion: 0.0.1
+        "401":
+          description: The currently supplied session does not have the correct access
```
Comment (dovholuknf) [inline client.yml:339]: "currently supplied session" is redundant. "The session does not have..."

File: management.yml [hunk 7]
```diff
@@ -3629,6 +3641,98 @@ paths:
...
+  /controllers:
+    get:
+      security:
+      - ztSession: []
+      - oauth2:
+        - openid
+      description: Retrieves a list of controllers
+      tags:
+      - Controllers
+      summary: List CAs
```
Comment (dovholuknf) [inline management.yml:3653]: List CAs again here

File: management.yml [hunk 8]
```diff
@@ -3629,6 +3641,98 @@ paths:
...
+                    fooField2: def
+                causeMessage: schema validation failed
+                code: COULD_NOT_VALIDATE
+                message: The supplied request contains an invalid document
+                requestId: ac6766d6-3a09-44b3-8d8a-1b541d97fdd9
+              meta:
+                apiEnrollmentVersion: 0.0.1
+                apiVersion: 0.0.1
+        "401":
+          description: The currently supplied session does not have the correct access
```
Comment (dovholuknf) [inline management.yml:3700]: same redundant 'currently supplied' thing as before

File: client.yml (hunk 4 above)
Comment (andrewpmartinez) [reply client.yml]: Fixed...all of them.

**Review by dovholuknf — APPROVED** [review]

File: client.yml (hunk 6 above)
Comment (andrewpmartinez) [reply client.yml]: Fixed....it is a shared string so it is updated in lots of places.

**Pull request merged**